calyx/src/lang/structure.rs

```diff
@@ -52,6 +52,8 @@ impl NodeData {
 
     pub fn out_ports(&self) -> PortIter {
         match self {
+            //XXX(Zhijing): wired to have input here
+            // but node data requires exhaustive pattern matching
```

Comment:
I noticed this on my verilog branch too. should be `PortIter { items: vec![] }` for the `Input` case.